readme.md

```diff
@@ -108,7 +109,7 @@ Configure it in `package.json`.
 - [no-unused-properties](docs/rules/no-unused-properties.md) - Disallow unused object properties.
 - [prefer-node-append](docs/rules/prefer-node-append.md) - Prefer `append` over `appendChild`. *(fixable)*
 - [prefer-query-selector](docs/rules/prefer-query-selector.md) - Prefer `querySelector` over `getElementById`, `querySelectorAll` over `getElementsByClassName` and `getElementsByTagName`. *(partly fixable)*
-
```

Comment:
Any reason to remove this line? :smirk:

Reply:
This is just some git stuff, i guess.

Reply:
Each header in the readme is padded with 2 blank lines, I think it aid's it's readability, and is worth preserving.

Reply:
sorry, but I can't see any difference.
https://github.com/WillDonut/eslint-plugin-unicorn/blob/add-prefer-includes-rule/readme.md
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/readme.md